pkg/policy/distillery.go

```diff
@@ -139,6 +139,8 @@ func (cache *PolicyCache) LocalEndpointIdentityRemoved(identity *identityPkg.Ide
 
 // Lookup attempts to locate the SelectorPolicy corresponding to the specified
 // identity. If policy is not cached for the identity, it returns nil.
+//
+//nolint:staticcheck // Because this causes a spurious error in endpoint/policy.go
```

Comment:
Kinda unfortunate that we need to apply nolint here as well. Does this effectively disable all linting in this function?

Reply:
It's super frustrating, especially because the "lint error" at this line is only emitted as an informational bit to explain a (spurious) issue at a caller. But, the tooling isn't smart enough to ignore the informational hint and fails lint without the tag here as well.

This disables the staticcheck linter for this function. Given that it's a one-line trivial function, it's not a big deal. Once the heat is off, I'm going to try and come up with a minimal reproducer and file a bug.

Reply:
edit: the lint error doesn't show up if the variable in question is a structure field (maybe because it can't determine shared access?) so these tags can be removed, but not for a good reason.